chain/config/config.go

```diff
@@ -13,6 +13,9 @@ type TestChainConfig struct {
 
 	// CheatCodeConfig indicates the configuration for EVM cheat codes to use.
 	CheatCodeConfig CheatCodeConfig `json:"cheatCodes"`
+
+	// ContractAddressOverrides describes contracts that are going to be deployed at deterministic addresses
+	ContractAddressOverrides map[common.Hash]common.Address `json:"contractAddressOverrides,omitempty"`
```

Comment:
It looks like this is always initialized in `fuzzer.go` and shouldn't hit a nil deref but maybe `omitempty` could be left off?

Reply:
so omitempty was intentional because I don't want this configuration parameter to be shown in the config file (`medusa.json`). This is because the user must update `Fuzzing.PredeployedContracts` which is then parsed and then used to update this variable. Thus, showing both may confuse the end user.